fe/lib/components/LocationSuggest/components/MapSelect.tsx

```diff
@@ -0,0 +1,176 @@
+import { useQuery } from '@apollo/client';
+import {
+  Alert,
+  Box,
+  ButtonIcon,
+  Column,
+  Columns,
+  IconClear,
+  Stack,
+  Text,
+  useColor,
+  useResponsiveValue,
+} from 'braid-design-system';
+import { Map, Marker, Overlay, ZoomControl } from 'pigeon-maps';
+import React from 'react';
+import { useState } from 'react';
+import { InlineCode } from 'scoobie';
+import { useDebounce } from 'use-debounce';
+
+import { Location } from '../../../types/seekApi.graphql';
+import { formatPoint } from '../../../utils/formatPoint';
+
+import { NEAREST_LOCATIONS } from './queries';
+
+import * as styles from './styles.css';
+
+const DEFAULT_ZOOM = 10;
+
+interface Props {
+  debounceDelay?: number;
+  schemeId: string;
+  onLocationSelected: (location: Location) => void;
+  initialLocation: [number, number];
+}
+
+export const MapSelect = ({
+  debounceDelay = 250,
+  schemeId,
+  onLocationSelected,
+  initialLocation,
+}: Props) => {
+  const [center, setCenter] = useState(initialLocation);
+  const [latLong, setLatLong] = useState(initialLocation);
```

Comment:
I haven't dug into the code too deeply, but what's the difference between these 2? It seems like they could be used interchangeably

Reply:
In the old design one was used as a state to keep track of and set the center of the map, whilst the other was used to control the position of the marker. This allowed you to create a marker at a single lat long, and then set the center separately, e.g. clicking on a marker brought the maps centre to it. In the new design I have it so that the map centres on each marker placement, meaning that we can just use one of these instead of both. I have removed the `centre` state and kept the `latLong` state 👍